
uint8 UnityScene
    uint8 SPACESHIP_INTERIOR=0
    uint8 DESTROYED_CITY=1
    uint8 INDUSTRIAL_PARK=2
    uint8 POLYGON_CITY=3
    uint8 DESERT_MOUNTAIN=4

uint8 RacetrackSite
    uint8 SITE_A=0
    uint8 SITE_B=1
    uint8 SITE_C=2

uint8 RacetrackType
    uint8 FIGURE8_DETERMINISTIC=0
    uint8 FIGURE8_RANDOMIZED=1
    uint8 CIRCUIT_RANDOMIZED=2
    uint8 SPRINT_RANDOMIZED=3
    uint8 INTERMEDIATE_TARGET_LOSS=4

bool RacetrackClockwise

uint8 RacetrackMode
    uint8 NARROW_GAP=0
    uint8 WIDE_GAP=1


# old names
uint8 RaceType
bool DynGatesActivated
    uint8 FIG8_DET=0
    uint8 FIG8_RAND=1
    uint8 CIRCUIT_RAND=2
    uint8 SPRINT_RAND=3

uint8 RaceGateType
    uint8 RPG_GATE=0
    uint8 TUB_DAI_GATE=1
    uint8 THU_DME_GATE=2


uint8 RANDOM_CHOICE=255
---

geometry_msgs/Pose DroneInitPose
geometry_msgs/Pose[] GatesInitPose #=Waypoints

# old names
geometry_msgs/Pose Drone_InitPose
geometry_msgs/Pose[] Gates_WaypointPose